src/app/components/client/ProgressCard.tsx

```diff
@@ -57,8 +60,16 @@ export const ProgressCard = (props: Props) => {
             style={activeProgressBarStyle}
           ></div>
           <div className={styles.percentageBreakdown}>
-            <p>{percentageCompleteNum}% complete</p>
-            <p>{percentageRemainingNumber}% in progress</p>
+            <p>
+              {l10n.getString("progress-card-percentage-complete", {
+                percentage: `${percentageCompleteNum}%`,
+              })}
```

Comment:
@flodolo Re: percentage formats, does this work or do we need to let localizers tweak the "%" sign?

I did a quick check of `Intl.NumberFormat(.., { style: "percent" }).format(...)` [for all locales with at least 70% translations in Pontoon] and it seems almost all locales use `nn%` or `nn %` and [`tr` (Turkish)](https://pontoon.mozilla.org/tr/firefox-monitor-website/all-resources/?search=%25&status=translated&string=206995) was the only one that might invert the sign to the left side of the number.



<details>
<summary>RESULTS</summary>

```js
// Via `npx pdehaan/pontoonql firefox-accounts 70 | jq '[.[] | .locale.code]'`
const locales = [
  "cs",  "en-GB",  "fr",  "de",  "it",
  "pt-BR",  "sr",  "es-AR",  "es-CL",
  "sv-SE",  "is",  "fi",  "fy-NL",  "da",
  "ru",  "nl",  "be",  "vi",  "zh-CN",
  "sk",  "sl",  "uk",  "ja",  "es-ES",
  "pl",  "ia",  "zh-TW",  "hu",  "cy",
  "gn",  "tr",  "en-CA",  "sq",  "kk",
  "hsb",  "ka",  "el",  "es-MX",  "he",
  "rm",  "pa-IN",  "nn-NO",  "kab",
  "dsb",  "pt-PT",  "ko"
].sort();

for (const locale of locales) {
  console.log(`[${locale}]\t${format(locale, 0.76)}`);
}

function format(locale, value = 0) {
  return new Intl.NumberFormat(locale, {
    style: "percent",
  }).format(value);
}
```

LOCALE | PERCENT
:----:|----:
be | 76 %
cs | 76 %
cy | 76%
da | 76 %
de | 76 %
dsb | 76 %
el | 76%
en-CA | 76%
en-GB | 76%
es-AR | 76 %
es-CL | 76 %
es-ES | 76 %
es-MX | 76%
fi | 76 %
fr | 76 %
fy-NL | 76%
gn | 76%
he | 76%
hsb | 76 %
hu | 76%
ia | 76%
is | 76%
it | 76%
ja | 76%
ka | 76%
kab | 76%
kk | 76%
ko | 76%
nl | 76%
nn-NO | 76 %
pa-IN | 76%
pl | 76%
pt-BR | 76%
pt-PT | 76%
rm | 76 %
ru | 76 %
sk | 76 %
sl | 76 %
sq | 76%
sr | 76%
sv-SE | 76 %
tr | %76
uk | 76%
vi | 76%
zh-CN | 76%
zh-TW | 76%

</details>

Reply:
This doesn't work, if nothing else because of the spacing (NEVER hard-coded characters…).

Reply:
Yes.

```
progress-card-percentage-complete = { $percentage }% complete
```